src/main/scala/chisel3/Driver.scala

```diff
@@ -149,7 +149,11 @@ object Driver extends BackendCompilationUtilities {
     /* create custom transforms by finding the set of transform classes associated with annotations
      * then instantiate them into actual transforms
      */
-    val transforms = circuit.annotations.map(_.transform).toSet.map { transformClass: Class[_ <: Transform] =>
+    val transforms = circuit.annotations
+                            .map(_.transform)
+                            .distinct
+                            .filterNot(_ == classOf[firrtl.Transform])
```

Comment:
Is this something you want to throw a warning / error with helpful message on instead of just silently dropping?